### ISSUE TITLE
No name is provided for Edit fields present in the task screen: A11y_.NET 9 template_.NET 9 template_Proejct_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26819]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value; #MAC;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Project section and press enter.
4. Project screen will open, observe that No name is provided for edit fields present in the project screen.

**Similar issue is also observed for categories and tags fields**

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter, tab till manage meta and press enter on it, Categories and tags screen will open.
4. Observe that No name is provided for edit fields present in the categories and tags screen.

### Actual Result:
No name is provided for Edit fields present in the Task screen.

### Expected Result:
Appropriate Name should be provided for Edit fields present in the Task screen.

### User Impact:
Screen reader users will get the confuse if no name is provided for edit fields available.

### Attachment:

https://github.com/user-attachments/assets/e2433fd5-d0ff-412a-915b-00f9f045c4c1

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/ae6e2535-c35b-4ed0-92b3-42096984088f" />

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/503d8f20-9c34-4405-80b3-432857616b9a" />

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/ea98cd17-e614-4355-ae02-4d82c176c640" />

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/351e8c75-57d8-4ef7-adbc-3f46df3605fc" />

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/6a9ef44a-22c1-4b52-aab2-c3ab65214937" />

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by AnKushSingh05** [comment]
Issue is fixed

<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/0785112b-1a91-44e1-b6f0-99f593340645" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/46a24a81-c585-45af-b9d8-73e607333aad" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/f9e4c396-90c9-479c-9136-aa5ae3d6e3a1" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/6690558b-5652-45a0-8ed2-e7eb8ecb83e2" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/fac3f06f-7c94-4867-b1b4-074dc1657e28" />
